src/plasma/plasma_client.c

```diff
@@ -250,13 +253,18 @@ void plasma_get(plasma_connection *conn,
     *data = lookup_mmapped_file(conn, object->handle.store_fd);
   } else {
     /* Else, request a reference to the object data from the plasma store. */
-    plasma_request req = plasma_make_request(object_id);
-    CHECK(plasma_send_request(conn->store_conn, PLASMA_GET, &req) >= 0);
-    plasma_reply reply;
-    CHECK(plasma_receive_reply(conn->store_conn, sizeof(reply), &reply) >= 0);
+    CHECK(plasma_send_GetRequest(conn->store_conn, conn->builder, &obj_id, 1) >= 0);
+    uint8_t *reply_data = plasma_receive(conn->store_conn, MessageType_PlasmaGetReply);
+    object_id* gotten_object_id;
+    int64_t num_objects;
+    plasma_read_GetReply(reply_data, &gotten_object_id, &object_data, &num_objects);
+    free(reply_data);
+    DCHECK(num_objects == 1);
+    DCHECK(memcmp(gotten_object_id, &obj_id, UNIQUE_ID_SIZE) == 0);
```

Comment:
Can we replace all instances of `UNIQUE_ID_SIZE` with `sizeof(obj_id)`, or whatever the appropriate thing is?